src/pool/srv_iv.c

```diff
@@ -873,6 +873,16 @@ pool_iv_ent_refresh(struct ds_iv_entry *entry, struct ds_iv_key *key,
 		D_GOTO(out_put, rc = 0);
 	}
 
+	if (entry->iv_key.eph > key->eph && key->eph != 0) {
```

Comment:
can key->eph == 0?

Reply:
yes, only IV update (sync & update) will set eph, for fetch eph is 0, and this callback will be called for fetch as well, so we just ignore this if key_eph is 0.